AppointEase/AppointEase.AspNetCore/Controllers/AuthController.cs

```diff
@@ -0,0 +1,85 @@
+﻿using AppointEase.Application.Contracts.Identity;
+using AppointEase.Application.Contracts.Interfaces;
+using AppointEase.Application.Contracts.Models;
+using AppointEase.Application.Contracts.Models.DbModels;
+using AppointEase.Application.Contracts.ModelsDto;
+using AppointEase.Application.Services;
+using AppointEase.Data.Repositories;
+using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace AppointEase.AspNetCore.Controllers
+{
+    public class AuthController : Controller
```

Comment:
Why is this called Auth? Are we authenticating something? Are we creating something?

Reply:
If this Controller will be used strictly for creating a Patient than should be called `PatientController`, with endpoints named HttpPost["Create"]..